docs/samples/client/kfserving_sdk_v1beta1_sample.ipynb

```diff
@@ -22,7 +22,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from kubernetes import client \n",
+    "from kubernetes import client\n",
```

Comment:
Next time let's separate these changes into a separate PR which can make PR review easier

Reply:
Somehow I cannot avoid it (haven't figured out why yet). These changes happen automatically by black+pre-commit and the build would fail without including the changes.